modules/TestUtils.js

```diff
@@ -8,6 +8,32 @@ export const isFunction = (object) =>
   typeof object === 'function'
 
 /**
+ * Returns true if the given object is an array.
+ */
+export const isArray = (object) =>
+  Array.isArray(object)
+
+/**
+ * Returns true if the given object is an object.
+ */
+export const isObject = (object) =>
+  Object.prototype.toString.call(object) === '[object Object]'
```

Comment:
This won't be reliable in an environment that supports ES6's Symbol.toStringTag. Is there a reason this isn't just `object && !isArray(object) && typeof object === 'object'`?